pyface/ui/wx/action/action_item.py

```diff
@@ -74,7 +74,7 @@ def __init__(self, parent, menu, item, controller):
         action  = item.action
         label   = action.name
         kind    = _STYLE_TO_KIND_MAP[action.style]
-        longtip = action.description
+        longtip = action.description or action.tooltip
```

Comment:
This is a change in semantics. If we want to do this, then the same logic should be applied on the `qt4` side as well.